### ISSUE TITLE
can't install 3.0.1 with pip install

### ISSUE THREAD

**Issue by mircealungu** [flask-dashboard/Flask-MonitoringDashboard#254]
**Describe the bug**
pip install -r requirements.txt from Zeeguu-API

**To Reproduce**
```
Collecting flask_monitoringdashboard>=3.0.1 (from -r requirements.txt (line 9))
  Downloading https://files.pythonhosted.org/packages/11/2a/4620e448d7bab55c662c8f97df56efd14fbe6b4418da8af7684f81cf0e84/Flask-MonitoringDashboard-3.0.1.tar.gz (207kB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-zuhwz3oy/flask-monitoringdashboard/setup.py", line 34, in <module>
        long_description=get_description(),
      File "/tmp/pip-install-zuhwz3oy/flask-monitoringdashboard/setup.py", line 10, in get_description
        with open(loc + '/docs/README.rst') as readme:
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-install-zuhwz3oy/flask-monitoringdashboard/docs/README.rst'

```

**Comment by mircealungu** [comment]
seems that the README.rst did not get published ?

**Comment by mircealungu** [comment]
fixed in d26f045637a108aac820d9bd0e0218b41b8897a4
